libraries/AP_RCTelemetry/AP_CRSF_Telem.h

```diff
@@ -19,7 +19,7 @@
 #include <AP_OSD/AP_OSD.h>
 
 #ifndef HAL_CRSF_TELEM_ENABLED
-#define HAL_CRSF_TELEM_ENABLED !HAL_MINIMIZE_FEATURES
+#define HAL_CRSF_TELEM_ENABLED (!HAL_MINIMIZE_FEATURES && AP_FRSKY_SPORT_PASSTHROUGH_ENABLED)
```

Comment:
I think this is wrong - you can have CRSF telemetry without passthrough

Reply:
Uh - it really does look like we get both the frsky and the passthrough singleton....